1/Dockerfile.rhel7

```diff
@@ -42,10 +44,10 @@ RUN yum install -y git tar zip unzip java-1.7.0-openjdk jenkins-1.609.1 && \
 
 COPY ./contrib/openshift /opt/openshift
 COPY ./contrib/jenkins /usr/local/bin
+ADD ./contrib/s2i /usr/libexec/s2i
 
 RUN chmod -R og+rw {/opt/openshift,/var/lib/jenkins}
```

Comment:
we should switch this to chgrp root since that's our pattern for enabling access now.